lua/lspconfig/server_configurations/stylelint_lsp.lua

```diff
@@ -7,6 +7,37 @@ if vim.fn.has 'win32' == 1 then
   cmd = { 'cmd.exe', '/C', bin_name, '--stdio' }
 end
 
+local function fix_all(opts)
+  opts = opts or { sync = true, bufnr = 0 }
+  local bufnr = util.validate_bufnr(opts.bufnr or 0)
+
+  local stylelint_lsp_client = util.get_active_client_by_name(bufnr, 'stylelint_lsp')
+  if stylelint_lsp_client == nil then
+    return
+  end
+
+  local request
+  if opts.sync then
```

Comment:
This kind of thing is why in https://github.com/neovim/nvim-lspconfig/issues/1937 I'm very hesitant to add these special-case commands to each lspconfig. It's unpleasant for users to have a different interface or every random command. Here we have a function with an "async" flag. Looks like it's always true currently. But this is a lot of code, and only for one command. I don't see how we can maintain 100s of configs with this much code.

Reply:
for the record I don't like either of these commands and I think both are workarounds.

That being said, I just want to fix my stylelint errors on save. I have no idea why `autoFixOnSave` and `autoFixOnFormat` are not respected, but this config and an aucmd will will solve the problem for users.

Reply:
@bennypowers 
```lua
require('lspconfig')['stylelint_lsp'].setup({
    settings = {
      stylelintplus = {
        autoFixOnFormat = true,
      },
    },
})
```
plus lukas-reineke/lsp-format.nvim seems to be fixing stylelint errors for me.

Reply:
Thanks for the link, but this pr is for support in lspconfig without an additional plugin

I'm definitely going to check that out for my own config tho